typings/storybook__addons/index.d.ts

```diff
@@ -47,7 +47,7 @@ declare module '@storybook/addons' {
        * Allows to override code that will be used for "Show Code" tab.
        * @see https://github.com/storybookjs/storybook/blob/main/addons/docs/docs/recipes.md#customizing-source-snippets
        */
-      transformSource: (snippet: string, story: StoryContextForEnhancers) => string | undefined;
+      transformSource?: (snippet: string, story: StoryContextForEnhancers) => string | undefined;
```

Comment:
this is not a required property - was causing TS errors once public-docsite started to have proper type-checking